src/applications/auth/components/HelpdeskContact.jsx

```diff
@@ -1,16 +0,0 @@
-import React from 'react';
-import { CONTACTS } from '@department-of-veterans-affairs/component-library/contacts';
-
-export default function Helpdesk({ children }) {
-  const startSentence = !children?.length;
-
-  return (
-    <p>
-      {children}
-      {startSentence ? 'Please' : ' please'} call our MyVA411 main information
-      line for help at <va-telephone contact={CONTACTS.HELP_DESK} /> (TTY:
```

Comment:
There's some slight differences between the `ContactCenterInformation` text and the old `HelpdeskContact` text. I'm assuming the `ContactCenterInformation` text is preferred?

Reply:
I left a note in the PR description asking that. The ticket says we want to keep IdentityPhone across the board and just rename it so I went with that one.